src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MovieType = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<MovieType> = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard movie={movie} />
```

Comment:
add key attribute

Reply:
in the task it is not necessary to change the html tags